blog-post.html

```diff
@@ -1,12 +1,161 @@
+          repellendus aliquam quasi ducimus unde fugiat dignissimos, earum voluptates dolores et dolor debitis quo
+          facere ullam, architecto hic! Aut sapiente sunt nemo, quas rem, blanditiis ea fuga ratione quis exercitationem
+          eveniet unde harum minima quod iusto ab eum cumque? Corporis quae saepe ipsa minima laboriosam ut neque amet
+          dolorem iure reiciendis fuga nobis deleniti perspiciatis odio, veniam necessitatibus suscipit hic vitae eum id
+          architecto at beatae! Nulla rerum tempora, dolorum nemo libero at temporibus voluptate sequi rem quis
+          laboriosam voluptas qui. Ad ut hic numquam, voluptatem asperiores voluptas sed!
+        </p>
+      </div>
+    </div>
+  </section>
+  <section id="footer">
+    <h2 class="header">this is the footer header</h2>
+    <div id="submit-form">
+      <!-- form -->
+      <form action="/my-handling-form-page" method="post">
+        <input type="text" id="name" name="user_name" placeholder="Name">
+        <input type="email" id="mail" name="user_email" placeholder="Email">
+        <textarea id="msg" name="user_message" placeholder="Message"></textarea>
+        <button type="button"><img src="/images/noun_Email_1053359.svg"> <p>Submit</p></button>
+      </form>
+      <div id="social-media-stuff">
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolor soluta nihil voluptatem molestias odio quos
+          est? Ullam aliquam consectetur expedita nihil facere adipisci itaque assumenda facilis impedit sint esse saepe
+          totam, corporis, fuga qui eaque. Illum, dolore aperiam quo omnis accusamus reiciendis, repudiandae ipsam
+          molestiae incidunt ullam magnam! Accusamus, harum.</p>
+        <div id="left">
+          <div>
+            <img src="images/noun_Address_1595349.svg">
+            <p>48 Monroe st. <br>
+              Burlington, vt <br>
+              5041</p>
+          </div>
+          <div>
+            <img src="images/noun_Phone_373695.svg">
+            <a href="tel:+xxxxxxxxxx">
+              <p>xxx-xxx-xxxx</p>
+            </a>
+          </div>
+          <div>
+            <img src="images/noun_Email_1053359.svg">
+            <a href="mailto:info@untitled.com">
+              <p>info@untitled.com</p>
+            </a>
+          </div>
+        </div>
+        <div id="right">
+          <div>
+            <img src="images/noun_Twitter_101100.svg">
+            <p>twitter</p>
+          </div>
+          <div>
+            <img src="images/noun_Camera_3318729.svg">
+            <p>instagram.com</p>
+          </div>
+          <div>
+            <img src="images/noun_Basketball_2824796.svg">
+            <p>dribble.com</p>
+          </div>
+          <div>
+            <img src="images/noun_thumbs up_683037.svg">
+            <p>facebook.com</p>
+          </div>
+        </div>
+      </div>
+    </div>
+

```

Comment:
This is super close to the example provided, you could scale the font size down slightly on the right side to have it match a little bit closer, or you can make your text area larger so that the right side and left side are the same height. 

Also setting some space between your left hand column on the right hand side of this would make it a closer match as well